yellowbrick/features/manifold.py

```diff
@@ -120,30 +111,45 @@ class Manifold(FeatureVisualizer):
         embedding. If n_neighbors is not specified for those embeddings, it is
         set to 5 and a warning is issued. If the manifold algorithm doesn't use
         nearest neighbors, then this parameter is ignored.
-
-    colors : str or list of colors, default: None
-        Specify the colors used, though note that the specification depends
-        very much on whether the target is continuous or discrete. If
-        continuous, colors must be the name of a colormap. If discrete, then
-        colors can be the name of a palette or a list of colors to use for each
-        class in the target.
-
-    target : str, default: "auto"
+        
+    features : list, default: None
+        a list of feature names to use
+        If a DataFrame is passed to fit and features is None, feature
+        names are selected as the columns of the DataFrame.
+
+    classes : list, default: None
+        a list of class names for the legend
+        If classes is None and a y value is passed to fit then the classes
+        are selected from the target vector.
+
+    colors : list or tuple of colors, default: None
+        Specify the colors for each individual class.
+
+    colormap : string or cmap, default: None
+        Optional string or matplotlib cmap to colorize points.
+        Use either color to colorize the points on a per class basis or
+        colormap to color them on a continuous scale.
+        
+    target_type : str, default: "auto"
         Specify the type of target as either "discrete" (classes) or "continuous"
-        (real numbers, usually for regression). If "auto", the Manifold will
+        (real numbers, usually for regression). If "auto", then it will
         attempt to determine the type by counting the number of unique values.
 
-        If the target is discrete, points will be colored by the target class
-        and a legend will be displayed. If continuous, points will be displayed
-        with a colormap and a color bar will be displayed. In either case, if
-        no target is specified, only a single color will be drawn.
+    projection : int or string, default: 2
+        The number of axes to project into, either 2d or 3d. To plot 3d plots
+        with matplotlib, please ensure a 3d axes is passed to the visualizer,
+        otherwise one will be created using the current figure.
 
-    alpha : float, default: 0.7
+    alpha : float, default: 0.75
         Specify a transparency where 1 is completely opaque and 0 is completely
         transparent. This property makes densely clustered points more visible.
 
     random_state : int or RandomState, default: None
         Fixes the random state for stochastic manifold algorithms.
+    
+    colorbar : bool, default: True
+        If the target_type is "continous" draw a colorbar to the right of the
+        scatter plot. The colobar axes is accessible using the cax property.
```

Comment:
Could we also please add `features_` to the `Attributes` section of the docstring?

Reply:
Sure!